ste/jobStatusManager.go

```diff
@@ -56,19 +61,73 @@ func (jm *jobMgr) SendXferDoneMsg(msg xferDoneMsg) {
 }
 
 func (jm *jobMgr) ListJobSummary() common.ListJobSummaryResponse {
-	jm.jstm.listReq <- true
-	return <-jm.jstm.respChan
+	if !jm.jstm.doneHandleStatusUpdateManager {
```

Comment:
comment above this

// Query job summary from handleStatusUpdateManager, but only if it's running.